### ISSUE TITLE
Feature - Text to QR code generator

### ISSUE THREAD

**Issue by Harishwarrior** [nileshtrivedi/devtoolbox#15]
Add Text to QR generator. Currently, we mostly use random websites to generate QR codes. 
@nileshtrivedi What do you think?

**Comment by nileshtrivedi** [comment]
@Harishwarrior Yes, that would be a good tool. https://github.com/gchq/CyberChef includes tools for generating and parsing QR codes. So, if we could fix #4 , we would get this feature for free. But that is blocked by some limitations in the Flutter platform.

**Comment by Harishwarrior** [comment]
> @Harishwarrior Yes, that would be a good tool. https://github.com/gchq/CyberChef includes tools for generating and parsing QR codes. So, if we could fix #4 , we would get this feature for free. But that is blocked by some limitations in the Flutter platform.
> 

This is a good tool. But, we can build those tools in pure dart.
